image/assets/lang/English.json

```diff
@@ -639,6 +639,17 @@
   "app_bell_onboarding_info_deep_click_warning": "<text weight='light' size='38'>You've </text><text weight='regular' size='38'>deep pressed</text>",
   "app_bell_onboarding_info_deep_click_correction": "<text weight='light' size='38'>Be more gentle,<br></br>try </text><text weight='regular' size='38'>light click </text><text weight='light' size='38'>this time</text>",
   "app_bell_onboarding_welcome": "Welcome",
+  "app_bell_onboarding_shortcuts_question": "Do you want to see Shortcuts instruction?",
+  "app_bell_onboarding_shortcuts_start": "Start",
+  "app_bell_onboarding_shortcuts_skip": "Skip",
+  "app_bell_onboarding_shortcuts_step_rotate": "Rotate to select",
+  "app_bell_onboarding_shortcuts_step_light_click": "Light click to accept and move forward",
+  "app_bell_onboarding_shortcuts_step_deep_press": "Deep press to active or deactivate the alarm",
+  "app_bell_onboarding_shortcuts_step_home": "Long press back (2s) to return to the home view",
+  "app_bell_onboarding_shortcuts_step_battery": "Long press back (2s) on the home view to see status battery",
+  "app_bell_onboarding_shortcuts_step_turn_off": "Press back for 10s to turn off the device",
+  "app_bell_onboarding_shortcuts_step_restart": "Press both side buttons for 10s to restart the device",
+  "app_bell_onboarding_shortcuts_step_bedside_lamp": "Long press frontlight (2s) to turn on a bedside lamp",
```

Comment:
what about translations for other languages?

Reply:
Good point. Other languages are part of this epic but will be delivered later as a different commit.